src/fiber/network.rs

```diff
@@ -3086,17 +3125,34 @@ where
             debug!("Tentacle service shutdown");
         });
 
+        let persistent_state = self
+            .store
+            .get_network_actor_state(&my_peer_id)
+            .unwrap_or_default();
+        let mut peers_to_connect = persistent_state.get_peers_to_connect();
```

Comment:
any special reason to put `persistent_state` in `NetworkActorState` but not in `NetworkActor`? since in other actors like the `ChannelActor`, we use a `store` for persistent storage and `ChannelActorState` don't have a field for `persistent_state`.

Reply:
This is because most fields in the network actor state are volatile, I add a persistent state to make the fact that these fields should be persisted more explicit.

Reply:
I feel the name `PersistentNetworkActorState` kind of sucks. It is actually the `NetworkActorStateRequiringPersistence`.